diff --git a/ihp-sg13g2/libs.tech/klayout/tech/lvs/sg13g2_full.lylvs b/ihp-sg13g2/libs.tech/klayout/tech/lvs/sg13g2_full.lylvs
index 86c8955..8a9614d 100644
--- a/ihp-sg13g2/libs.tech/klayout/tech/lvs/sg13g2_full.lylvs
+++ b/ihp-sg13g2/libs.tech/klayout/tech/lvs/sg13g2_full.lylvs
@@ -4657,6 +4657,41 @@ schematic.purge_nets if PURGE_NETS
 max_res(1e9)
 min_caps(1e-18)
 
+# === EQUIVALENT STANDARD CELL PINS ===
+equivalent_pins("sg13g2_a21o_1", "A1", "A2")
+equivalent_pins("sg13g2_a21o_2", "A1", "A2")
+equivalent_pins("sg13g2_a21oi_1", "A1", "A2")
+equivalent_pins("sg13g2_a21oi_2", "A1", "A2")
+equivalent_pins("sg13g2_a221oi_1", "A1", "A2")
+equivalent_pins("sg13g2_a221oi_1", "B1", "B2")
+equivalent_pins("sg13g2_a22oi_1", "A1", "A2")
+equivalent_pins("sg13g2_a22oi_1", "B1", "B2")
+equivalent_pins("sg13g2_and2_1", "A", "B")
+equivalent_pins("sg13g2_and2_2", "A", "B")
+equivalent_pins("sg13g2_and3_1", "A", "B", "C")
+equivalent_pins("sg13g2_and3_2", "A", "B", "C")
+equivalent_pins("sg13g2_and4_1", "A", "B", "C", "D")
+equivalent_pins("sg13g2_and4_2", "A", "B", "C", "D")
+equivalent_pins("sg13g2_nand2_1", "A", "B")
+equivalent_pins("sg13g2_nand2_2", "A", "B")
+equivalent_pins("sg13g2_nand3_1", "A", "B", "C")
+equivalent_pins("sg13g2_nand4_1", "A", "B", "C", "D")
+equivalent_pins("sg13g2_nor2_1", "A", "B")
+equivalent_pins("sg13g2_nor2_2", "A", "B")
+equivalent_pins("sg13g2_nor3_1", "A", "B", "C")
+equivalent_pins("sg13g2_nor3_2", "A", "B", "C")
+equivalent_pins("sg13g2_nor4_1", "A", "B", "C", "D")
+equivalent_pins("sg13g2_nor4_2", "A", "B", "C", "D")
+equivalent_pins("sg13g2_o21ai_1", "A1", "A2")
+equivalent_pins("sg13g2_or2_1", "A", "B")
+equivalent_pins("sg13g2_or2_2", "A", "B")
+equivalent_pins("sg13g2_or3_1", "A", "B", "C")
+equivalent_pins("sg13g2_or3_2", "A", "B", "C")
+equivalent_pins("sg13g2_or4_1", "A", "B", "C", "D")
+equivalent_pins("sg13g2_or4_2", "A", "B", "C", "D")
+equivalent_pins("sg13g2_xnor2_1", "A", "B")
+equivalent_pins("sg13g2_xor2_1", "A", "B")
+
 # === COMPARISON === 
 logger.info('Starting SG13G2 LVS Comparison')
 compare
